test/verifiedTests/analysis/testExploration/testFindRxnsFromGenes.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testFindRxnsFromGenes.m
+%
+% Purpose:
+%     - tests that reactions are found when providing a list of genes.
+%
+% Authors:
+%     - Original file: Stefania Magnusdottir August 2017
+%
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testFindRxnsFromGenes'));
+cd(fileDir);
+
+% load model
+load('Test_EcoliCoreModel.mat')
```

Comment:
Ah, this is supposed to use the ecoli_core_model.mat. I had to save the model locally for the test since my Matlab kept crashing when I used initCobraToolbox and using CBTDIR without first running initCobraToolbox returned CBTDIR = []. I've changed the test to use CBTDIR and get the E. coli core model from the test/models folder.